frontera/contrib/backends/remote/codecs/json.py

```diff
@@ -101,52 +135,51 @@ def __init__(self, request_model, response_model, *a, **kw):
         super(Decoder, self).__init__(*a, **kw)
 
     def _response_from_object(self, obj):
-        url = to_native_str(obj[b'url'])
+        url = obj['url']
         request = self._request_model(url=url,
```

Comment:
If url had type 'bytes' then after decoding it will have type bytes, and in Frontera URLs have to be native strings (when in process).